pixl_imaging/tests/docker-compose.yml

```diff
@@ -85,7 +85,6 @@ services:
             ORTHANC_ANON_AE_TITLE: "unused"
             ORTHANC_ANON_DICOM_PORT: "4242"
```

Comment:
mentioned it before but might be worth setting restart to always

Reply:
ah yep, good point